modules/mono/glue/GodotSharp/GodotSharp/Core/Transform3D.cs

```diff
@@ -207,6 +226,19 @@ public Transform3D Scaled(Vector3 scale)
             return new Transform3D(basis.Scaled(scale), origin * scale);
         }
 
+        /// <summary>
+        /// Scales the transform by the given 3D <paramref name="scale"/> factor.
+        /// The operation is done in the local frame, equivalent to
+        /// multiplying the matrix from the right.
+        /// </summary>
+        /// <param name="scale">The scale to introduce.</param>
+        /// <returns>The scaled transformation matrix.</returns>
+        public Transform3D ScaledLocal(Vector3 scale)
+        {
+            Basis tmpBasis = new Basis(new Vector3(scale.x, 0, 0), new Vector3(0, scale.y, 0), new Vector3(0, 0, scale.z))
```

Comment:
CI is complaining about this line, missed a semicolon.

```suggestion
            Basis tmpBasis = new Basis(new Vector3(scale.x, 0, 0), new Vector3(0, scale.y, 0), new Vector3(0, 0, scale.z));
```

Reply:
Thanks for the hint, I just realized the mistake as well ;)